### ISSUE TITLE
After today, it blocks showing my calendars and my events on Calendar

### ISSUE THREAD

**Issue by ChristianSino** [imightbeamy/gcal-multical-event-merge#93]
I've never had an issue so far. But after today, when enabled, it blocks loading the calendar list and the events on the Calendar and it keeps loading it indefinitely. Is it the latest Chrome updates? I don't know.

**Comment by HCAWN** [comment]
Hi Chistian, perhaps my fork of this extension that I've been updating regularly fixes this issue?
https://chromewebstore.google.com/detail/cal-merge-for-google-cale/dchpdbljalbfnelbfbgncfidhjoibkah

**Comment by ChristianSino** [comment]
> Hi Chistian, perhaps my fork of this extension that I've been updating regularly fixes this issue? https://chromewebstore.google.com/detail/cal-merge-for-google-cale/dchpdbljalbfnelbfbgncfidhjoibkah

Yeah, I've already been using this extension instead! Thank you for developing and supporting it.
